sigle/src/modules/settings/plans/ComparePlans.tsx

```diff
@@ -0,0 +1,298 @@
+            }}
+          />
+          <Typography size="h4" css={{ fontWeight: 600 }}>
+            Starter
+          </Typography>
+          <Typography size="subheading" css={{ color: '$gray9' }}>
+            All the basic
+          </Typography>
+          <Typography size="h4" css={{ fontWeight: 600 }}>
+            Free
+          </Typography>
+          <Typography
+            size="subheading"
+            css={{ backgroundColor: '$green5', px: '$3', py: '$1', br: '$2' }}
+          >
+            Current plan
+          </Typography>
+        </Th>
+        <Th
+          css={{
+            display: 'flex',
+            flexDirection: 'column',
+            gap: '$2',
+            alignItems: 'center',
+            backgroundColor: '$gray2',
+            maxWidth: 220,
+          }}
+          scope="col"
+        >
+          <Box
+            css={{
+              borderRadius: '20px 20px 0 0',
+              height: 8,
+              width: '100%',
+              backgroundColor: '$violet11',
+            }}
+          />
+          <Typography size="h4" css={{ fontWeight: 600 }}>
+            Creator +
+          </Typography>
+          <Typography size="subheading" css={{ color: '$gray9' }}>
+            Unlock with your NFT
+          </Typography>
+          <Typography size="h4" css={{ fontWeight: 600 }}>
+            Free
+          </Typography>
+          <Button
+            onClick={() => setIsSelectNFTDialogOpen(true)}
+            size="lg"
+            color="violet"
+            css={{ width: 'calc(100% - $6)' }}
+          >
+            Link your NFT
+          </Button>
+        </Th>
+      </Tr>
+    </thead>
+    <tbody>{children}</tbody>
+  </StyledTable>
+);
+
+export const ComparePlans = () => {
+  const [isSelectNFTDialogOpen, setIsSelectNFTDialogOpen] = useState(false);
+
+  const getFeatureStatus = (value: PlanStatus) => {
+    switch (value) {
+      case 'active':
+        return <CheckIcon />;
+      case 'inactive':
+        return null;
+      case 'progress':
+        return '⚙️';
+      default:
+        throw new Error('No value received.');
+    }
+  };
+
+  return (
+    <SettingsLayout>
+      <Link href="/settings/plans">
+        <Button css={{ gap: '$2', cursor: 'pointer' }} variant="subtle" as="a">
+          <ArrowLeftIcon />
+          Go back to your current plan
+        </Button>
+      </Link>
+      <Table setIsSelectNFTDialogOpen={setIsSelectNFTDialogOpen}>
+        {features.map((feature) => (
+          <Tr
+            css={{
+              '&:last-of-type': {
+                '& td': {
+                  br: '0 0 20px 20px',
+                  pb: '$8',
+                },
+              },
+            }}
+            key={feature.name}
+          >
+            <Th
+              css={{
+                display: 'flex',
+                flexDirection: 'column',
+                justifyContent: 'center',
+                maxWidth: 335,
```

Comment:
manually setting maxWidth here (same for `Td` and `Th` elements) to adjust width according to design, as I couldn't manage to get a working solution with flex or grid